core/include/drivers/imx_mu.h

```diff
@@ -13,10 +13,10 @@
 #define IMX_MU_DATA_U16(mesg, idx) ((mesg)->data.u16[(idx)])
 #define IMX_MU_DATA_U8(mesg, idx)  ((mesg)->data.u8[(idx)])
 
-#define IMX_MU_MSG_SIZE	  7
+#define IMX_MU_MSG_SIZE	  17
```

Comment:
No need for a `CFG_MX93` conditional here?

Reply:
Nope. I've updated `IMX_MU_MSG_SIZE` so the MU can hold the biggest ELE message, which is 17 words long ... But yeah, I will move this change to a dedicated commit since this is not related to imx93 support.

Reply:
Sounds good

Reply:
Maybe call it `IMX_MAX_MU_MSG_SIZE`?

Reply:
I prefer to keep `IMX_MU_MSG_SIZE` :)

Reply:
No problem